worker/codalabworker/local_run/local_reader.py

```diff
@@ -102,7 +102,9 @@ def read_file_section(self, run_state, path, dep_paths, args, reply_fn):
         """
 
         def read_file_section_thread(final_path):
-            bytestring = gzip_string(read_file_section(final_path, args['offset'], args['length']))
+            bytestring = gzip_bytestring(
+                read_file_section(final_path, args['offset'], args['length'])
```

Comment:
I'm assuming this was the piece of code that was failing before this PR, correct?

Reply:
Yeah.